### DISCUSSION TITLE
더블클릭을 1회 클릭으로 바꿀지 논의

### DISCUSSION THREAD

**Discussion by juyoungpark718** [Team-Panopticon/bookmark-dashboard#54] [General]

**Comment by junsushin-dev** [comment]
1안 -> 클릭
장점: Web UX와 유사
단점: 추후 multi-drag 기능 지원 시 불편할 수 있음

2안 -> 더블클릭
장점: 바탕화면 UX와 유사
단점: 추후 multi-drag 기능 추가 시 유리함

**Comment by junsushin-dev** [comment]
<2022-02-23>
현재 다중선택, 드래그 기능이 없기 때문에 클릭으로 투표해서 결과 정함
